code/modules/recycling/disposal.dm

```diff
@@ -103,62 +119,97 @@
 			return
 		if((S.allow_quick_empty || S.allow_quick_gather) && length(S.contents))
 			S.hide_from(user)
-			user.visible_message("[user] empties \the [S] into \the [src].", "You empty \the [S] into \the [src].")
+			user.visible_message(
+				"<span class='notice'>[user] empties [S] into the disposal unit.</span>",
+				"<span class='notice'>You empty [S] into disposal unit.</span>",
+				"<span class='notice'>You hear someone emptying something full of smaller objects out into a disposal unit.</span>"
+				)
 			for(var/obj/item/O in S.contents)
 				S.remove_from_storage(O, src)
 			S.update_icon() // For content-sensitive icons
 			update()
 			return
 
+	// Borg using their gripper to throw stuff away.
+	if(istype(I, /obj/item/gripper/))
+		var/obj/item/gripper/gripper = I
+		// Gripper is empty.
+		if(!gripper.gripped_item)
+			to_chat(user, "<span class='warning'>You're not holding anything!</span>")
```

Comment:
```suggestion
			to_chat(user, "<span class='warning'>There's nothing in your gripper to throw away!</span>")
```
this better fulfills the purpose of this message. Why is the game telling me there's nothing in my gripper? Oh, it's because it thinks I'm trying to throw something away.